src/controllers/krate/versions.rs

```diff
@@ -302,4 +354,228 @@ struct PaginatedVersionsAndPublishers {
 struct ResponseMeta {
     total: i64,
     next_page: Option<String>,
+    release_tracks: Option<ReleaseTracks>,
```

Comment:
might make sense to use `skip_if_empty` (or whatever it is called) for this field

Reply:
Good call!